common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -220,6 +220,10 @@ public String get(String key, final String defaultValue) {
     return updated;
   }
 
+  public String get(String key) {
+    return get(key, null);
```

Comment:
Should this be changed to not take default value and just throw exception if it doesn't exist?